README.Rmd

```diff
@@ -36,6 +36,8 @@ We encourage contributions on any part of the book, including:
 
 Please see [style.md](https://github.com/Robinlovelace/geocompr/blob/master/style.md) for the book's style.
 
+If you would like to add to the references, please join Zotero and add your citation to the open [geocompr group library](https://www.zotero.org/groups/418217/energy-and-transport/items/collectionKey/9K6FRP6N).
```

Comment:
Heads-up @Nowosad I think it will be useful to have a shared place to manage citations so I set this up as an open library: https://www.zotero.org/groups/418217/energy-and-transport/items/collectionKey/9K6FRP6N

Please contribute to it there if you want to add new citations. Any issues, let me know - can always add them manually but find Zotero (in combination with **citr**) the most efficient way for dealing with references.